benchmarks/benchmarks/stats.py

```diff
@@ -57,45 +57,107 @@ def time_ttest_ind_diff_var(self):
 
 
 class Distribution(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that are files under `distcont` and `distdiscrete`.
+    dists = sorted(list(set([d[0] for d in distcont + distdiscrete])))
+
     param_names = ['distribution', 'properties']
     params = [
-        ['cauchy', 'gamma', 'beta'],
-        ['pdf', 'cdf', 'rvs', 'fit']
+        dists, ['pdf/pmf', 'logpdf/logpmf', 'cdf', 'logcdf', 'rvs', 'fit',
+                'sf', 'logsf', 'ppf', 'isf', 'moment', 'stats_s', 'stats_v',
+                'stats_m', 'stats_k', 'stats_mvsk', 'entropy']
     ]
+    distcont = dict(distcont + distdiscrete)
+    # maintain previous benchmarks' values
+    custom_input = {'gamma': [5], 'beta': [5, 3]}
```

Comment:
Is `cauchy` not included because it used the same shapes as in `distcont`?

Reply:
Cauchy has no shape